context/response/responses.go

```diff
@@ -0,0 +1,53 @@
+package response
+
+import (
+	beecontext "github.com/astaxie/beego/context"
+)
+
+// JSON renders value to the response as JSON
+func JSON(value interface{}, encoding ...bool) Renderer {
```

Comment:
StatusNotFound is simply a const - I don't think it should be placed on the context object.
Redirect returns a renderer. It does not changes directly the output - so I don't think it belongs on the Output object itself. also c.Ctx.Output.Redirect feels to verbose IMO